changes/bug4700

```diff
@@ -0,0 +1,5 @@
+  o Minor features (onion services):
+    - Next generation onion services can now use the per-service
```

Comment:
I would say we should use "verson 3" instead of "next gen".

Reply:
Agreed

Reply:
✓